source/_components/media_player.enigma2.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Enigma2"
+description: "Instructions on how to integrate an Enigma2 based box running OpenWebif into Home Assistant."
+date: 2019-02-21 12:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: openwebif.png
+ha_category: Media Player
+featured: false
+ha_release: 0.90
+ha_iot_class: "Local Polling"
+---
+
+The `enigma2` platform allows you to control a Linux based set-top box which is running [Enigma2](https://github.com/oe-alliance/oe-alliance-enigma2) with the OpenWebif plugin installed.
+
+[OpenWebif](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif) is an open source web interface for Enigma2 based set-top boxes.
```

Comment:
Are there limitations which devices it can be installed? Worth mentioning here? Is the setup so straightforward, that there is no need to describe how to set it up here?

Reply:
No, there are no limitations. In fact, you will struggle to buy an Enigma2 based box in the past 15 years which does not come pre-installed with OpenWebif. There is literally no setup involved for the end user. It is enabled on port 80 by default with no auth.